mindmaps-graql/src/main/java/io/mindmaps/graql/internal/query/ComputeQueryImpl.java

```diff
@@ -68,7 +65,11 @@ public Object execute() throws ExecutionException, InterruptedException {
                 return analytics.degrees();
             }
             case "degreesAndPersist": {
```

Comment:
This was just to get analytics in quickly. We're eventually going to support custom "compute" methods, which will probably go in some kind of map.